styles/Navbar.module.css

```diff
@@ -0,0 +1,52 @@
+.nav {
+    display: flex;
+    flex-direction: row;
+    justify-content: center;
+    align-items: center;
+    padding: 0px;
+    position: absolute;
+    left: 84px;
+    top: 54px;
```

Comment:
<img width="766" alt="Screen Shot 2021-08-28 at 11 17 32" src="https://user-images.githubusercontent.com/18134219/131227167-db31fbfc-0e23-4da2-bc98-bfbd3518c5f0.png">

i think having `position: absolute` takes the navbar out of the _page flow_.. the screenshot hopefully shows what i mean by this. it may be possible to achieve the same formatting using `margin`

Reply:
Very true - I didn't look at that. Fixed it to margins!